test/utils/test.utils.js

```diff
@@ -0,0 +1,15 @@
+import fastify from 'fastify';
+import { initAppModule } from '../../src/app.module.js';
+import { config } from '../../src/config.js';
+
+export const buildApp = (opts = {}) => {
```

Comment:
if there are default options - mention them in the comments at least. If not, why is there an empty object?